### PR TITLE
Standardize apt/apt-get calls in tests and messages

### PR DESCRIPTION
**Pull request by renanrodrigo**
## Why is this needed?
This PR solves all of our problems because now we use `apt` instead of `apt-get` to simulate user interaction with the client in the acceptance tests.
We also don't suggest that people run `apt-get` in our hints anymore.

## Test Steps
mostly a refactor, integration tests cover the changes messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in:

## Does this PR require extra reviews?
 - [ ] Yes
 - [x] No

### CODE REVIEW

**Comment by renanrodrigo** [comment]
>     * [x]  Existing messages are being changed.
>       
>       * ⚠️ Please add a comment with justification of why messages are being altered.

That's the spirit of the first commit - fix messages.

>       * If the changes are trivial (e.g. a typo fix), then translations must be preserved.

They are

File: features/steps/packages.py [hunk 1]
```diff
@@ -3,19 +3,91 @@
...
+@when("I install `{package_names}`")
+@when("I install `{package_names}` on the `{machine_name}` machine")
+def when_i_apt_install(context, package_names, machine_name=SUT):
+    when_i_retry_run_command(
         context,
         " ".join(
             [
                 "DEBIAN_FRONTEND=noninteractive",
-                "apt-get",
+                "apt",
```
Comment (lucasmoura) [inline features/steps/packages.py:90]: Should we also add  `-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" ` here ?

**Review by lucasmoura — APPROVED** [review]
Just a question on the install step, but let me know if that is not needed

File: features/steps/packages.py (hunk 1 above)
Comment (renanrodrigo) [reply features/steps/packages.py:90]: We didn't in any of the steps in the current tests (thus it is not here) but I think it does not hurt. Will add.

Comment (orndorffgrant) [reply features/steps/packages.py:90]: Those are already here :)

File: features/steps/packages.py [hunk 2]
```diff
@@ -3,19 +3,91 @@
...
+        cmd_list.append("-o APT::Get::Always-Include-Phased-Updates=true")
+
+    when_i_run_command(
+        context,
+        " ".join(cmd_list),
+        "with sudo",
+    )
+
+
+@when("I install `{package_names}`")
```
Comment (orndorffgrant) [inline features/steps/packages.py:82]: I had it as "I apt install" to differentiate it from a snap install, do you think that is a useful differentiation?

File: features/steps/packages.py [hunk 3]
```diff
@@ -37,6 +37,39 @@ def when_i_apt_update(context, machine_name=SUT):
     )
 
 
+@when("I perform a {dry} run of {command} upgrade")
+@when("I upgrade all apt packages in the system")
+@when("I upgrade all apt packages in the system using {command}")
+@when("I upgrade all apt packages in the system including {phased} updates")
```
Comment (orndorffgrant) [inline features/steps/packages.py:52]: Do we ever not want to include phased updates?

**Comment by orndorffgrant** [comment]
I'm just a little hesitant to endorse the step names, mostly because I selfishly don't want to memorize new phrases for these :upside_down_face: 
We could name them more closely to the commands like:
```
When I apt upgrade
When I apt dist-upgrade
When I apt remove `package1 package2`
```
Let's talk about it in 2024 :)

File: features/steps/packages.py (hunk 3 above)
Comment (renanrodrigo) [reply features/steps/packages.py:52]: I would say the standard is to not include them, so we can consider acceptance tests to do what users do.
We install phased updates when testing our apt messages, for practicality purposes.

File: features/steps/packages.py (hunk 1 above)
Comment (renanrodrigo) [reply features/steps/packages.py:90]: AHA

File: features/steps/packages.py (hunk 2 above)
Comment (renanrodrigo) [reply features/steps/packages.py:82]: yes

**Review by orndorffgrant — APPROVED** [review]
ci fails unrelated

**Pull request merged**